### ISSUE TITLE
Stats printing: emit oversized allocs / sec

### ISSUE THREAD

**Issue by davidtgoldblatt** [jemalloc/jemalloc#2141]
Clearing out some old notes, and found a reminder to do this: it would be handy to emit the number of oversized (and #/sec). This should be straightforwardly obtainable from existing stats facilities, but it's not easy to see at a glance as a human.

**Comment by davidtgoldblatt** [comment]
I think that'd be useful too; I think the main motivation was to be able to tell, at a glance, "wow, this program is doing a whole lot more >8MB allocations than I'd expect". That's the kind of thing that would jump out at you right away, whereas just having the arena id would require you to guess that that would be the problem and go looking for it.

**Comment by interwq** [comment]
Makes sense. It doesn't hurt to show both. To clarify, you are thinking about the human-readable format of stats_print, right?
